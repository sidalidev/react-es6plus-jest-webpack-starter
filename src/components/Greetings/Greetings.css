p {
  color: peru;
  font-family: Helvetica, sans-serif;
  font-size: 15vw;
}
