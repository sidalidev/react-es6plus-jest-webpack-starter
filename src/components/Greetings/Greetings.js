import React, { Component } from 'react'
import './Greetings.css'

export default class Greetings extends Component {
    render() {
        return <p>Hello There!</p>
    }
}
