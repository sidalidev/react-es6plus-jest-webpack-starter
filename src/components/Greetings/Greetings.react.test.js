import React from 'React'
import Greetings from './Greetings'
import renderer from 'react-test-renderer'

test('Should greet', () => {
    const component = renderer.create(<Greetings />)

    let tree = component.toJSON()
    expect(tree).toMatchSnapshot()
})
