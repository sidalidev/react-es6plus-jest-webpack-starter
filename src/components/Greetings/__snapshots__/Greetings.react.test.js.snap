// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should greet 1`] = `
<p>
  Hello There!
</p>
`;
